core/backend/src/SqliteStatement.ts

```diff
@@ -11,21 +11,23 @@ import { ECJsNames, IModelError } from "@bentley/imodeljs-common";
 import { IModelJsNative } from "@bentley/imodeljs-native";
 import { IModelHost } from "./IModelHost";
 
-/** Marks a string as either an [Id64String]($bentleyjs-core) or [GuidString]($bentleyjs-core), so
- * that it can be passed to the [bindValue]($backend.SqliteStatement) or [bindValues]($backend.SqliteStatement)
- * methods of [SqliteStatement]($backend).
- * @internal
+/** an object returned by [[SqliteStatement.getRow]]
+ * @public
  */
-export interface StringParam {
-  id?: Id64String;
-  guid?: GuidString;
+export interface RowValue {
+  [propName: string]: Uint8Array | number | string;

```

Comment:
`| undefined` for null?

Reply:
we don't add it to the result if it's null, apparently.

Reply:
Right but if I do `SELECT a, b FROM c` and get back `row`, if a is null then `row.a` will be undefined. Type should account for that.

Reply:
I'm always confused about the difference between "not present" and "undefined". Does `typeof row.a` return `"undefined"` if there's no `a` in `row`?

Reply:
Yes. `undefined` means "no value".

Reply:
added